README.Rmd

```diff
@@ -83,6 +83,34 @@ library(bcdata)
 - [Exploring Silviculture Data with bcdata](https://bcgov.github.io/bcdata/articles/explore-silviculture-data-using-bcdata.html)
 - Using bcdata with [bcmaps](https://github.com/bcgov/bcmaps) (Coming Soon!)
 
+### BCDC Authentication
+
+If you are an authorized user of the B.C. Data Catalogue you may want to access
```

Comment:
I think everyone are authorized users, perhaps it is more clear to use `If you are an authorized editor of the BCDC?`

Reply:
Good calll